src/sass/components/footer.sass

```diff
@@ -112,5 +111,54 @@
       border-width: 6px 6px 0 6px
       border-color: $sub-text transparent transparent transparent
 
+// Language switcher
+.lang-switch-link
+  display: inline-block
+  position: relative
+  transition: all 0s ease
+  svg
+    display: inline-block
+    fill: $sub-text
+    margin-bottom: .15em
+    width: 1em
+    height: 1em
+    vertical-align: middle
+  .footer-country-select__arrow
+    width: .75em
+    height: .75em
+    margin-bottom: .185em
+
+.lang-switch-link:hover
+  .footer-country-select__arrow
+    transform: rotate(180deg)
```

Comment:
You could add: `transition-delay: .3s` in this case popover and icon will rotate/appear together.